src/main/java/com/three_tech_solutions/slot_app/services/implementations/PriceServiceImpl.java

```diff
@@ -20,14 +20,14 @@ public class PriceServiceImpl implements PriceService {
 
     private final PriceRepository priceRepository;
     @Override
-    public PriceUpdateResponse updatePriceAmount(UUID priceId, PriceUpdateRequest request) {
-        Price price = priceRepository.findById(priceId)
+    public PriceResponse updatePriceAmount(UUID priceId, PriceUpdateRequest request) {
+        return priceRepository.findById(priceId)
+                .map(price -> {
+                    price.setAmount(request.amount());
+                    priceRepository.save(price);
+                    return PriceMapper.toPriceUpdateResponse(price);
```

Comment:
Siguiendo el comentario anterior, acá tendriamos que agregar esa dependencia con el mapper


```suggestion
private final PriceRepository priceRepository;
private final PriceMapper priceMapper;

@Override
    public PriceResponse updatePriceAmount(UUID priceId, PriceUpdateRequest request) {
        return priceRepository.findById(priceId)
                .map(price -> {
                    price.setAmount(request.amount());
                    priceRepository.save(price);
                    return priceMapper.toPriceUpdateResponse(price);
```